tests/contrib/dbapi/test_dbapi.py

```diff
@@ -31,6 +31,18 @@ def test_execute_wrapped_is_called_and_returned(self):
         assert "__result__" == traced_cursor.execute("__query__", "arg_1", kwarg1="kwarg1")
         cursor.execute.assert_called_once_with("__query__", "arg_1", kwarg1="kwarg1")
 
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_DBM_PROPAGATION_MODE="full"))
+    def test_dbm_propagation_not_supported(self):
+        cursor = self.cursor
+        cfg = IntegrationConfig(Config(), "dbapi", service="dbapi_service")
+        # By default _dbm_propagation_supported attribute should not be True.
+        # DBM context propagation should be opt in.
+        assert not getattr(cfg, "_dbm_propagation_supported", False)
```

Comment:
```suggestion
        # By default _dbm_propagation_supported attribute should be false.
        # DBM context propagation should be opt in.
        assert not getattr(cfg, "_dbm_propagation_supported")
```

Reply:
This will raise an AttributeError. `cfg._dbm_propagation_supported` isn't always defined. This is a pattern we use in database integrations to add attributes to the integration config object ([example](https://github.com/DataDog/dd-trace-py/blob/v1.6.0/ddtrace/contrib/psycopg/patch.py#L27)).  Should we do `assert not getattr(cfg, "_dbm_propagation_supported", None)` instead?